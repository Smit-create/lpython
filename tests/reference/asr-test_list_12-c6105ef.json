{
    "basename": "asr-test_list_12-c6105ef",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/test_list_12.py",
    "infile_hash": "2d4c3d12a0c8bada168530947680b4a61b0d9bfd30ce0ca474127aa3",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-test_list_12-c6105ef.stdout",
    "stdout_hash": "fc7ff2108d8b8df461598be74603cf0c4c9854972b33a0f62131de3d",
    "stderr": "asr-test_list_12-c6105ef.stderr",
    "stderr_hash": "d213c8a5c229f3851ceea0907fe60125b96fc321a5739726166c577b",
    "returncode": 0
}