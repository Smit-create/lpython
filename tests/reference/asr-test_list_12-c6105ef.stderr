warning: `List` is a deprecated alias to `list`
 --> tests/../integration_tests/test_list_12.py:4:20
  |
4 |     WHITESPACE   : List[str] = [" ", ",", "ok", "\r", "\t"]
  |                    ^^^^ Hint: Use `list` here
